components/Navbar/HelpLink.tsx

```diff
@@ -36,7 +36,7 @@ export default function HelpLink({ params }: { params: any }) {
           target="_blank"
           href="https://forms.gle/NVcg9ukPvUBYjw1u7"
         >
-          ?
```

Comment:
By the way, the "?" should be Nunito and not Caveat.

Reply:
Hey, @GBKS I am not sure about that  '?'. The only line code which I added is related to a screen reader. 
Also, I would like to ask about the title of the power-off button. Right now I changed it to 'back to chapter selection page'. Please let me know what's your opinion about it.

Reply:
Yep, Ben is correct. The new title also looks good.